recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-core/base-files/base-files_%.bbappend

```diff
@@ -9,4 +9,7 @@ do_install_append () {
 	echo "	shopt -s checkwinsize" >> ${D}${sysconfdir}/profile
 	echo "  resize 1> /dev/null" >> ${D}${sysconfdir}/profile
 	echo "fi" >> ${D}${sysconfdir}/profile
+	if [ ! -z "${AOS_VIS_PLUGINS}" ]; then
```

Comment:
Could anyone move this change to meta-prod-aos?

Reply:
@varder I can ceherry-pick it there, but it is needed only after migration to the new bsb will be merged into the master.

Reply:
@varder  or you whant to move it out from prod-devel ? if so - it is not possible.

Reply:
> @varder or you whant to move it out from prod-devel ? if so - it is not possible.

i think he meant to duplicate it , @varder , right ?